grantsAfterRemoveREAL.csv

```diff
@@ -0,0 +1,71 @@
+address,allocation,withdrawn,created,cliff,expiration,latestVotingPower,delegatee,rangeStart,rangeEnd
```

Comment:
@sentilesdal Can you add a comment explaining what the old CSV files (that were deleted) and the new CSV files were tracking and are tracking now?

Reply:
They are all snapshots of grants at different times.  I placed them in the out/ dir so they wouldn't be checked in anymore since there were too many of them to make sense.  The latest ones are tracking before and after the current proposal.  Not sure if you mean to add a comment here or elsewhere.